BetterPasswordSchemaMigration.php

```diff
@@ -15,26 +15,30 @@
 use Illuminate\Database\Migrations\Migration;
 use Illuminate\Database\Schema\Builder;
 use Illuminate\Database\Schema\Blueprint;
-//use Illuminate\Database\Capsule\Manager as Capsule;
-//use Illuminate\Support\Facades\DB;
 use Illuminate\Support\Facades\Schema;
 
 class BetterPasswordSchemaMigration extends Migration {
-		/**
-		 * Run the migrations.
-		 * @return void
-		 */
+	/**
+	 * Run the migrations.
+	 * @return void
+	 */
 	public function up() {
-		// Bad Password failed login attempts
-		//Capsule::schema()->create('badpw_failedlogins', function (Blueprint $table) {
 		Schema::create('badpw_failedlogins', function (Blueprint $table) {
 			$table->string('username', 32);
 			$table->bigInteger('count');
 			$table->datetime('failed_login_time');
 		});
+
+		Schema::create('stored_passwords', function (Blueprint $table) {
+			$table->bigIncrements('id');
+			$table->integer('user_id');
+			$table->text('password');
+			$table->datetime('last_change_time');
+		});
```

Comment:
As part of the migration, we will also need to move the settings from the User table to the new stored_passwords table.  Example 3.4 migrations can be found here:
https://github.com/pkp/pkp-lib/tree/1979b7a55eb1cdea648de423d06a6378dc7e5fc2/classes/migration/upgrade/v3_4_0